demo/admin/src/products/ManufacturerFilter.tsx

```diff
@@ -0,0 +1,72 @@
+import { gql, useQuery } from "@apollo/client";
+import Autocomplete from "@mui/material/Autocomplete";
+import TextField from "@mui/material/TextField";
+import { GridFilterInputValueProps, GridFilterOperator } from "@mui/x-data-grid-pro";
+import * as React from "react";
+import { useIntl } from "react-intl";
+
+import { GQLManufacturersFilterQuery, GQLManufacturersFilterQueryVariables } from "./ManufacturerFilter.generated";
+
+const manufacturersQuery = gql`
+    query ManufacturersFilter($offset: Int!, $limit: Int!, $search: String) {
+        manufacturers(offset: $offset, limit: $limit, search: $search) {
+            nodes {
+                id
+                name
+            }
+            totalCount
+        }
+    }
+`;
+
+// Source: https://mui.com/x/react-data-grid/filtering/customization/#multiple-values-operator
+function ManufacturerFilter({ item, applyValue }: GridFilterInputValueProps) {
+    const intl = useIntl();
+
+    const { data } = useQuery<GQLManufacturersFilterQuery, GQLManufacturersFilterQueryVariables>(manufacturersQuery, {
+        variables: {
+            offset: 0,
+            limit: 10,
+        },
+    });
+
+    // source https://mui.com/material-ui/react-autocomplete/
+    return (
+        <Autocomplete
+            size="small"
+            options={data?.manufacturers.nodes ?? []}
+            autoHighlight
+            value={item.value}
+            isOptionEqualToValue={(option, value) => {
+                // does only highlight the selected value in options-list but does not trigger getOptionLabel-Call
+                return option.id == value;
+            }}
+            getOptionLabel={(option) => {
+                // option.name is not set if loaded from filters-model, small delay replacing id with name because of loading
```

Comment:
Is this because the name isn't in the URL? Could this be changed?

Reply:
yes that's the problem. but I don't think it's the best idea to always require id+label to be referenced together... I was surprised async-select does not support fetching it's label on it's own, at least if required.

Reply:
This is due to the nature we use the select in forms in combination with GraphQL, where we don't need a request for the label because it's already defined by the form values. I don't like it much either, but I'm not sure if changing it to remove the label from the form state is a better solution...

Reply:
how would you load the label? that would require an extra graphql query

Reply:
yes, if needed it would also require referencing which query to use to get the label... but it's probably always possible to solve this label-"issue" with a workaround. so never mind :-D